pkg/v1/providers/ytt/vendir/kapp-controller/_ytt_lib/addons/packages/kapp-controller/0.33.1/bundle/config/values.star

```diff
@@ -14,5 +14,5 @@ def generateBashCmdForDNS(coreDNSIP):
     # In this way, Kapp Controller will have cluster IP access,
     # and still able to resolve enternal urls when core DNS is unavailable
 
-    return "cp /etc/resolv.conf /etc/resolv.conf.bak; sed '1 i nameserver " + coreDNSIP + "' /etc/resolv.conf.bak > /etc/resolv.conf; rm /etc/resolv.conf.bak; cp -R /etc/* /kapp-etc; chmod g+w /kapp-etc/pki/tls/certs/ca-bundle.crt && chgrp 2000 /kapp-etc/pki/tls/certs/ca-bundle.crt"
+    return "cp /etc/resolv.conf /etc/resolv.conf.bak; sed '1 i nameserver " + coreDNSIP + "' /etc/resolv.conf.bak > /etc/resolv.conf; rm /etc/resolv.conf.bak; cp -R /etc/* /kapp-etc; chmod g+w /kapp-etc/pki/tls/certs/ca-bundle.crt"
```

Comment:
@ridavid2002 I think this bug fix https://github.com/vmware-tanzu/community-edition/pull/3259/files  
is missed while adding 0.33.1 kapp-controller package in TCE. Could you open a PR in TCE to fix this. Thanks

Reply:
@aaronshurley Is the chgrp 2000 /kapp-etc/pki/tls/certs/ca-bundle.crt intentionally removed ?

Reply:
yep! This conversation might make things clearer on that decision: https://github.com/vmware-tanzu/community-edition/pull/3476#discussion_r823163912